packages/mailgun/src/models/v1/emails/request/override-properties/override-properties.ts

```diff
@@ -0,0 +1,34 @@
+export interface OverrideProperties {
+  /** Tag string.  See **Tagging** for more information */
+  tag?: string;
+  /** Toggles Send Time Optimization (STO) on a per-message basis.  String should be set to the number of hours in `[0-9]+h` format, with the minimum being `24h` and the maximum being `72h`.  This value defines the time window in which Mailgun will run the optimization algorithm based on prior engagement data of a given recipient.  See **Sending a Message with STO** for details.  *Please note that STO is only available on certain plans. See www.mailgun.com/pricing for more info* */
+  deliveryTimeOptimizePeriod?: string;
```

Comment:
Added value manage a simple integer, like what was done with conversation for `ttl` field, and serialized it according to server expectation : https://github.com/sinch/sinch-sdk-node/blob/main/packages/conversation/src/rest/v1/messages/messages-api.ts#L297